src/main/java/com/xfers/net/APIResource.java

```diff
@@ -88,4 +87,58 @@ public static String request(APIResource.RequestMethod method,
         }
 
     }
+
```

Comment:
instead of copying and pasting, can we have another method which both request and requestCOnnect calls? so theres no code duplicate?